.github/workflows/ci.yml

```diff
@@ -217,11 +217,16 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-18.04, macos-10.15, windows-2019]
+        go-version: ['1.16.3']
+        include:
+          # Go 1.13.x is still used by Docker/Moby
+          - go-version: '1.13.x'
```

Comment:
when 1.18 drops (generics) n-3 will become problematic..

Reply:
Sorry what do you mean?

Reply:
There is a language change for go called generics. It is currently planned for v1.18... if/when that happens we'll have to decide if we allow the use of said feature or if we will forego using it to help out the people who are not moving up.

Reply:
Ah, I was expecting generics to be in Go 2.0 👀 

We will have to help Moby-related repos to migrate to the latest Go before that...

Reply:
My impression is there won't be an actual go2.0 release, and the "go 2" project is more about driving the language forwards through 1.x releases.